src/main/java/pl/multiplex/multiplexreservation/api/ReservationApi.java

```diff
@@ -1,4 +1,69 @@
 package pl.multiplex.multiplexreservation.api;
 
-public class ReservationApi {
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import pl.multiplex.multiplexreservation.dao.entity.Movie;
+import pl.multiplex.multiplexreservation.dao.entity.Screening;
+import pl.multiplex.multiplexreservation.dao.entity.Seat;
+import pl.multiplex.multiplexreservation.dao.entity.User;
+import pl.multiplex.multiplexreservation.manager.ReservationManager;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("multiplex/reservation")
+public class ReservationApi
+{
+    private ReservationManager reservations;
+
+
+    @Autowired
+    public ReservationApi(ReservationManager reservation) {
+        this.reservations = reservation;
+    }
+
+    @GetMapping("/screening/all")
+    public Iterable<Screening> getAll() {return reservations.showAll();}
+
+    @GetMapping("/screening")
+    public List<Screening> getByDayAndTime(@RequestParam int date, int time) {
+        return reservations.showByTimeAndDay(date,time);
+    }
+
+    @GetMapping("/screening/interval")
+    public List<Screening> getByDayAndMinMaxTime(@RequestParam int date, int timeMin, int timeMax) {
```

Comment:
Czemu ta adnotacja jest tylko przy pierwszym parametrze?

Reply:
Z tutoriali zrozumiałem, że wystarczy tylko na początku listy argumentów. Wydawało mi się to poprawne.